### ISSUE TITLE
Zork files off tape deliberately corrupted

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1985]
@eswenson1 noticed several Zork files in LCF are corrupted.  They are ok in the git repository and they seem ok on the sources.tape.

The files we know are affected are, so far, ACT1 through ACT4 and DUNG.

**Comment by larsbrinkhoff** [comment]
I have confirmed the same files look bad on my TT ITS.  The strange thing is, another set of older files also have ACT1-4 and DUNG corrupted.

**Comment by larsbrinkhoff** [comment]
I tested this on an older ITS image I happened to have on my laptop.  The Zork files used to be in the ZORK director, and they were ok.  Then I made a tape with the LCF directory and loaded it.  The same files as above were corrupted.

**Comment by larsbrinkhoff** [comment]
I made a new tape with the Zork files in ZORK2, and they loaded fine.  I made another tape with the files in LCF2 and the usual suspects were garbled.

**Comment by larsbrinkhoff** [comment]
Files called ACT or DUN written to a directory called LCF-something are deliberately obfuscated by DUMP!

First check the directory name:

```
        SETZM HFNL
        HLRZ A,HSNM
        CAIN A,'LCF
         PUSHJ P,ISLCF
```

then file name:

```
ISLCF:  HLRZ A,HFN1
        CAIE A,'ACT
         CAIN A,'DUN
          SETOM HFNL
        POPJ P,
```

And if HFNL is set:

```
HFNLX:  PUSH P,B                ;DONT COMPROMISE, RANDOMIZE!
        PUSH P,A
HFNLXL: MOVE A,(B)
        TSC A,1(B)
        MOVEM A,(B)
        AOBJN B,HFNLXL
        POP P,A
        POP P,B
        POPJ P,
```

**Comment by larsbrinkhoff** [comment]
This code was first present in DUMP 254 timestamped 1979-02-15.  Previous version on record was DUMP 252 from 1978-11-01.

**Comment by larsbrinkhoff** [comment]
Fixed by #1988.
